astropy/time/tests/test_basic.py

```diff
@@ -2595,8 +2596,15 @@ def test_write_every_format_to_ecsv(fmt):
     t.write(out, format="ascii.ecsv")
     t2 = Table.read(out.getvalue(), format="ascii.ecsv")
     assert t["a"].format == t2["a"].format
-    # Some loss of precision in the serialization
-    assert not np.all(t["a"] == t2["a"])
+    if sys.version_info < (3, 13):
+        # Some loss of precision in the serialization
+        assert not np.all(t["a"] == t2["a"])
+    else:
+        # On 3.13.0b2, first tests indicate that loss
+        # of precision is not guaranteed.
```

Comment:
Is this good or bad?

Reply:
I think good ? But the test is a bit unusual to me, I don't think we want to _guarantee_ that serialization lead to loss of precision, do we ??

Reply:
Maybe @taldcroft can confirm since he added the test in #12842

Reply:
I do not understand how the updated test in this PR can be passing in 3.13. 

The test is asserting loss of precision if you take a time like `2001-01-01T00:00:00.14285714285714285` and then store it in ECSV as `2001-01-01T00:00:00.143` and use that formatted value to create the round trip Time object. This should always lose precision. The user can choose to store time in ECSV as JD using two doubles, in which case there is no loss, but that is a different test.

Reply:
thanks for checking, now I suspect there's a subtle bug somewhere in {CPython, numpy, us} . I'll spend more time digging into this !

Reply:
I report that the test pass without modification locally. So *something* is broken (most likely in the toolchain). Good ! this is what this PR is all about: finding bugs in our dependencies before it's too late to complain :)
beta 3 drops in 3 days (supposedly +1 or +2 to be available on GitHub Actions), so I'll try again then and if it still doesn't work, I'll dig into it.
In the mean time, let me revert this change here.